perun-cli/Perun/Agent.pm

```diff
@@ -143,6 +143,10 @@ sub call
 	my $fullUrl = "$self->{_url}/$format/$class/$method";
 
 	my $content = $self->{_jsonXs}->encode( $hash );
+
+	#print $content;
+	#print "\n\n\n";
```

Comment:
Please remove this debug part, or use it in more suitable way like "DEBUG=0; if(DEBUG) { print something; } if really needed.